symtabAPI/src/emitElf.C

```diff
@@ -1990,6 +1990,24 @@ bool emitElf<ElfTypes>::createSymbolTables(set<Symbol *> &allSymbols) {
     if (!obj->getAllNewRegions(newSecs))
         log_elferror(err_func_, "No new sections to add");
 
+    for (unsigned long nsi = 0; nsi < newSecs.size(); nsi++) {
+	int sec_addr, prev_size;
+	if (newSecs[nsi]->getDiskOffset())
+	  sec_addr = newSecs[nsi]->getDiskOffset() + library_adjust;
+	else
+	  sec_addr += prev_size;
```

Comment:
`prev_size` has no defined value here. It's also being updated on line 1999 in every iteration, so there is no lagging value.

Reply:
Have you had a chance to look at this?